bitnami/elasticsearch/templates/coordinating/statefulset.yaml

```
@@ -61,17 +62,17 @@ spec:
       {{- if .Values.coordinating.tolerations }}
       tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.coordinating.tolerations "context" $) | nindent 8 }}
       {{- end }}
+      {{- if .Values.coordinating.schedulerName }}
+      schedulerName: {{ .Values.coordinating.schedulerName }}
+      {{- end }}
       {{- if .Values.coordinating.topologySpreadConstraints }}
       topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.coordinating.topologySpreadConstraints "context" $) | nindent 8 }}
       {{- end }}
-      serviceAccountName: {{ template "elasticsearch.coordinating.serviceAccountName" . }}
-      {{- if or .Values.coordinating.podSecurityContext.enabled .Values.coordinating.securityContext.enabled }}
-      securityContext:
-        {{- if .Values.coordinating.podSecurityContext.enabled }}
-        {{- omit .Values.coordinating.podSecurityContext "enabled" | toYaml | nindent 8 }}
-        {{- else }}
-        fsGroup: {{ .Values.coordinating.securityContext.fsGroup }}
-        {{- end }}
+      {{- if .Values.coordinating.priorityClassName }}
+      priorityClassName: {{ .Values.coordinating.priorityClassName | quote }}
+      {{- end }}
```

Missing terminationGracePeriodSeconds:
```suggestion 
      {{- if .Values.coordinating.priorityClassName }}
      priorityClassName: {{ .Values.coordinating.priorityClassName | quote }}
      {{- end }}
      {{- if .Values.coordinating.terminationGracePeriodSeconds }}
      terminationGracePeriodSeconds: {{ .Values.coordinating.terminationGracePeriodSeconds }}
      {{- end }}
```